packages/st2/rpm/st2.spec

```
@@ -47,6 +47,12 @@ Conflicts: st2common
   %service_install st2scheduler
   make post_install DESTDIR=%{buildroot}
 
+# We build cryptography for EL8, and this can contain buildroot path in the
+# built .so files. We use strip on these libraries so that there are no
+# references to the buildroot in the st2 rpm
+%if 0%{?rhel} >= 8
```

What if we'll add RHEL 9 in the future?

This would be a more correct check: 
```suggestion
%if 0%{?rhel} == 8
```